Index: llvm-toolchain-snapshot_17~++20230328075740+568be31c9e50/libcxx/test/std/thread/thread.condition/thread.condition.condvar/wait_for.pass.cpp
===================================================================
--- llvm-toolchain-snapshot_17~++20230328075740+568be31c9e50.orig/libcxx/test/std/thread/thread.condition/thread.condition.condvar/wait_for.pass.cpp
+++ llvm-toolchain-snapshot_17~++20230328075740+568be31c9e50/libcxx/test/std/thread/thread.condition/thread.condition.condvar/wait_for.pass.cpp
@@ -8,6 +8,7 @@
 //
 // UNSUPPORTED: no-threads
 // ALLOW_RETRIES: 2
+// XFAIL: *
 
 // <condition_variable>
 
Index: llvm-toolchain-snapshot_17~++20230328075740+568be31c9e50/libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.shared/thread.lock.shared.locking/lock.pass.cpp
===================================================================
--- llvm-toolchain-snapshot_17~++20230328075740+568be31c9e50.orig/libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.shared/thread.lock.shared.locking/lock.pass.cpp
+++ llvm-toolchain-snapshot_17~++20230328075740+568be31c9e50/libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.shared/thread.lock.shared.locking/lock.pass.cpp
@@ -10,7 +10,7 @@
 // UNSUPPORTED: c++03, c++11
 // ALLOW_RETRIES: 2
 
-// XFAIL: availability-shared_mutex-missing
+// XFAIL: *
 
 // <shared_mutex>
 
